mkstage4.sh

```diff
@@ -80,7 +93,7 @@ shift;OPTIONS="$@"
 EXCLUDES="\
 --exclude=.bash_history \
 --exclude=dev/* \
---exclude=lost+found \
+--exclude=lost+found/* \
```

Comment:
Ah, good point. I was thinking in terms of a backup, in which case it might be a good to preserve the directory structure. But if the script is used to create a stage4 image for deploying on a separate machine (like the name would suggest), and that machine may have a different filesystem, then I agree it would not make sense to have the directory.